CanHazFunny/CanHazFunny/IDisplayJokes.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny;
+
+public interface IDisplayJokes
+{
+    public void OutputJoke(string joke);
```

Comment:
```suggestion
    void OutputJoke(string joke);
```
Interface members are public by default.